frontend/src/concepts/k8s/utils.ts

```diff
@@ -5,8 +5,10 @@ import { genRandomChars } from '~/utilities/string';
 export const PreInstalledName = 'Pre-installed';
 
 export const ownedByDSC = (resource: K8sResourceCommon): boolean =>
-  !!resource.metadata?.ownerReferences?.find((owner) => owner.kind === 'DataScienceCluster');
-
+  !!Object.prototype.hasOwnProperty.call(
+    resource.metadata?.labels,
+    'platform.opendatahub.io/part-of',
+  );
```

Comment:
```suggestion
  !!resource.metadata?.labels?.['platform.opendatahub.io/part-of'];
```
I imagine we can just do a straight-forward truthy check -- technically this would not work if the value is `''` but I don't think that's a valid situation.